.vsts-pr.yaml

```diff
@@ -61,10 +63,19 @@ jobs:
         publishLocation: Container
       continueOnError: true
       condition: not(succeeded())
+    - task: PublishBuildArtifacts@1
+      displayName: Publish test logs
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\TestResults\$(_configuration)'
+        FileCopyOptions: /xd cloudtest_*
```

Comment:
☹️ still haven't figured out how to make this work. Two copies of heap dumps are getting uploaded on failure

Reply:
I think xcopy options only work if `publishLocation: FilePath` is set instead of `container`